scalafix-core/shared/src/main/scala/scalafix/internal/rule/DottyVarArgPattern.scala

```diff
@@ -6,7 +6,8 @@ import scalafix.Patch
 import scalafix.rule.Rule
 import scalafix.rule.RuleCtx
 
-case object DottyVarArgPattern extends Rule("DottyVarArgPattern") {
+case object DottyVarArgPattern
+    extends Rule("DottyVarArgPattern", "Rule for Dotty new varargs syntax.") {
```

Comment:
Rewrite to convert `:_*` vararg pattern syntax to `@` syntax supported in Dotty.